Mlem/Views/Tabs/Profile/User View.swift

```diff
@@ -69,34 +61,48 @@ struct UserView: View {
             progressView
         }
     }
+    
+    @ViewBuilder
+    private var moderatorButton: some View {
+        if let user = userDetails {
+            if !moderatedCommunities.isEmpty {
```

Comment:
Generally I think it's cleaner to consolidate into a single check

`if let user = userDetails, !moderatedCommunities.isEmpty`

Since there's no else on the first condition

Reply:
Didn't even know that was possible, TIL!